modules/apps/object/object-service/src/main/java/com/liferay/object/internal/model/listener/ObjectEntryModelListener.java

```diff
@@ -164,7 +167,36 @@ private JSONObject _getPayloadJSONObject(
 			ObjectEntry objectEntry, long userId)
 		throws PortalException {
 
+		boolean sendInternalModel = GetterUtil.getBoolean(
+			PropsUtil.get(PropsKeys.WEBHOOK_EXPOSE_INTERNAL_MODEL));
+
 		return JSONUtil.put(
+			"internalObjectEntry",
```

Comment:
If we change the name of the JSON property we are still breaking the clients because they need to look for another property name.

Did you talk about this with Brian? Did he agree to rename the property?

I guess the problem is how to call the new property in a way that it makes sense and remains the same when the old property is deprecated and removed...

Reply:
> Did you talk about this with Brian? Did he agree to rename the property?

I'm afraid no. I did not talk about the renames with Brian. But I understand what you are saying. 
Should we propose a new name? Maybe I can think of a new name for the external model and maintain the old one for the internal model.